packages/ui/package-lock.json

```diff
@@ -5304,13 +5304,6 @@
         "lodash": "^4.17.0"
       }
     },
-    "node_modules/@gravity-ui/date-utils/node_modules/dayjs": {
```

Comment:
There is no changes in package.json file in the commit. Why do we have ones in package-lock.json?